File: leetcode2/1easy/이준열/3184.cpp
class Solution {
public:
    int countCompleteDayPairs(vector<int>& hours) {
        int output = 0;
        for (int i = 0; i < hours.size(); i++)
        {
            for (int j = i + 1; j < hours.size(); j++)
            {
                if((static_cast<long long>(hours[i]) + static_cast<long long>(hours[j])) % 24 == 0)
                    output++;
            }
        }
        return output;
    }
};

File: leetcode2/1easy/유시연/3184.py
class Leetcode.이재훈.이재훈.Solution:
    def countCompleteDayPairs(self, hours: List[int]) -> int:
        n = len(hours)
        count = 0

        for i in range(n):
            for j in range(n):
                if i <= j:
                    continue
                if (hours[i]+hours[j]) % 24 == 0:
                    count += 1


        return count